src/integrated/soil_canopy_model.jl

```diff
@@ -428,8 +421,11 @@ function lsm_radiant_energy_fluxes!(
 
     @. LW_u = (1 - ϵ_canopy) * LW_u_soil + ϵ_canopy * _σ * T_canopy^4 # double checked
 
-    # Effective (radiative) land properties 
-    @. p.α_sfc = SW_u / max(SW_d, eps(FT)) # TODO: replace with fraction reflected as compute by canopy
+    # Effective (radiative) land properties
+    # α = 1/(λ_max - λ_min) * ∫_λ_min ^λ_max α(λ) dλ
+    # We assume that the PAR and NIR bands are each exactly 50% of the incoming SW radiation
+    # Therefore the integral reduces to:
+    @. p.α_sfc = (f_abs_par + f_abs_nir) / 2
```

Comment:
This cannot be calculated like this. First, I think what you tried to do is: 
@. p.α_sfc = (f_refl_par + f_refl_nir) / 2

but this is not right. This is better: 
@. p.α_sfc = SW_u / max(SW_d, eps(FT))

Reply:
youre right! I meant f_refl_par, etc.

For the second comment - since SW_u is proportional to SW_d, if SW_d = 0, SW_u = 0, and albedo would not be defined (as you pointed out before). Because of that, in our current main branch, we have what you suggested. 

I thought this was not ok because then we would have albedo =0 at night, which would  mess up our monthly/daily averages that we save as output, and which also isnt exactly physical, since albedo should be a well defined physical property, and not depend on insolation. For example, the albedo of soil is always defined, even if light is not shining on it.

Reply:
So, albedo at night doesn't make sense anyway. So, it definitely shouldn't be included in the calculations of monthly, daily values. A quick workaround would be to set it to NaN when SW_d = 0. But if we need a number for the calculations, we can set it to zero when SW_d is zero.
Because while we do have this general assumption that PAR and NIR are roughy half of the energy, the fraction doesn't hold like this. So, it is best to keep track of SW_u and SW_d
Also, soil albedo and leaf albedo are intrinsic properties of the material, but canopy albedo is different, because it depends on the incident angle of radiation, the proportion of direct and diffuse, canopy structure, LAI, etc. So that is why we call leaf/soil albedo, single scattering albedo. 

That should be fine for the atmos coupling, because if there's no incident radiation SW 0 times 0 = 0. But for the diagnostic (daily, monthly averages), we will need to add NaNs instead. It is the same issue with the canopy temperature before. Otherwise we will have unrealistic low values of canopy albedo, driven by the zeros at night.

Reply:
This has been revert to what Renato suggested